linux_os/guide/system/software/integrity/software-integrity/rpm_verification/rpm_verify_permissions/rule.yml

```diff
@@ -16,7 +16,11 @@ description: |-
     <pre>$ rpm -qf <i>FILENAME</i></pre>
     <br />
     Next, run the following command to reset its permissions to the correct values:
+    {{% if 'rhel' in product %}}
+    <pre>$ sudo rpm --restore <i>PACKAGENAME</i></pre>
```

Comment:
I think in rpm_verify_permissions you need to update also the Bash and Ansible remediations.